controllers/storagecluster/reconcile.go

```diff
@@ -647,3 +672,24 @@ func validateArbiterSpec(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error
 	}
 	return nil
 }
+
+func setNetworkConfiguration(defaultRookConfigData string, val string) (string, error) {
```

Comment:
perhaps we can have a more general method like `updateRookConfig(config string, section string, value string )(string, error)`?

Reply:
@rohantmp what does the config and section string indicate?